apps/admin_api/lib/admin_api/v1/controllers/transaction_request_controller.ex

```diff
@@ -95,6 +95,30 @@ defmodule AdminAPI.V1.TransactionRequestController do
     |> respond(conn)
   end
 
+  @spec cancel(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def cancel(conn, %{"formatted_id" => formatted_id}) do
+    with {:ok, request} <- TransactionRequestFetcher.get(formatted_id) || {:error, :unauthorized},
```

Comment:
```suggestion
    with {:ok, request} <- TransactionRequestFetcher.get(formatted_id),
```

Remove unreachable condition